samples/cpp/tutorial_code/calib3d/real_time_pose_estimation/src/PnPProblem.cpp

```diff
@@ -20,7 +20,7 @@ cv::Point3f SUB(cv::Point3f v1, cv::Point3f v2);
 cv::Point3f get_nearest_3D_point(std::vector<cv::Point3f> &points_list, cv::Point3f origin);
 
 
-/* Functions for Möller–Trumbore intersection algorithm */
+/* Functions for Möller-Trumbore intersection algorithm */
```

Comment:
I think we'd better ask the authors of those files whether they want to change their names into English.